dockerfiles/backfill_storage/startup_script.sh

```diff
@@ -29,5 +29,5 @@ fi
 
 # TODO: update starting/ending block number to match extract diffs deploy block
 # Run getStorageValue
-echo "Running backfillStorage from block 0 to 0"
-./vulcanizedb backfillStorage -s=0 -e=0 --config config.toml
+echo "Running backfillStorage from block TBD to 11679900"
+./vulcanizedb backfillStorage -s=TBD -e=11679900 --config config.toml
```

Comment:
This starting block will need to be updated with around the block number where #440 was merged in.